feature-tests/run-tests.sh

```diff
@@ -33,6 +33,21 @@ echo "STACK_NAME ${STACK_NAME}"
 
 # run tests and save the exit code
 declare test_run_result
+
+if [ "${STACK_NAME}" != "local" ]; then
+
+  PARAMETERS_NAMES=(CORE_STUB_URL CORE_STUB_PASSWORD CORE_STUB_USERNAME FRONTEND)
+  tLen=${#PARAMETERS_NAMES[@]}
+   for (( i=0; i<${tLen}; i++ ));
+  do
+    echo "/tests/$STACK_NAME/${PARAMETERS_NAMES[$i]}"
+    PARAMETER=$(aws ssm get-parameter --name "/tests/$STACK_NAME/${PARAMETERS_NAMES[$i]}" --region eu-west-2)
+    VALUE=$(echo "$PARAMETER" | jq '.Parameter.Value')
+    NAME=$(echo "$PARAMETER" | jq '.Parameter.Name' | cut -d "/" -f4 | sed 's/.$//')
+
+    eval $(echo "export ${NAME}=${VALUE}")
+  done
+
```

Comment:
There is missing `fi` here to close the if

Reply:
Thanks Seamus, will update that now